scripts/dump_bin.py

```diff
@@ -476,9 +476,6 @@ def _dump_features(self):
                         futures[executor.submit(self._dump_bin, _df, _update_calendars)] = _code
                 else:
                     # new stock
-                    _dt_range = self._update_instruments.setdefault(_code, dict())
```

Comment:
This code will  act like
```
_dt_range = self._update_instruments[_code]
# and then do something on _dt_range 
```

Are you sure that it does nothing?

Reply:
@lerit

Reply:
sorry,my fault.